velox/substrait/proto/substrait/capabilities.proto

```diff
@@ -3,7 +3,6 @@ syntax = "proto3";
 
 package substrait;
 
-option cc_enable_arenas = true;
```

Comment:
Why do you remove this?

It would be nice to add velox/substrait/README.md file to document the procedure for upgrading Substrait. CC: @kgpai

Reply:
> Why do you remove this?

@mbasmanova Will fix this mistake, thanks

Reply:
@mbasmanova Done, please review again, thanks